roles/custom/matrix-bot-draupnir/defaults/main.yml

```diff
@@ -96,3 +105,25 @@ matrix_bot_draupnir_configuration_extension: "{{ matrix_bot_draupnir_configurati
 # Holds the final configuration (a combination of the default and its extension).
 # You most likely don't need to touch this variable. Instead, see `matrix_bot_draupnir_configuration_yaml`.
 matrix_bot_draupnir_configuration: "{{ matrix_bot_draupnir_configuration_yaml | from_yaml | combine(matrix_bot_draupnir_configuration_extension, recursive=True) }}"
+
+# Controls whether labels will be added that redirect the /_matrix/client/../rooms/../report paths to draupnir
+# Following these recommendations (https://github.com/element-hq/synapse/blob/master/docs/reverse_proxy.md), by default, we don't.
+# Regardless of whether this is enabled, it may or may not take effect due to the value of other variables.
+# See `matrix_synapse_container_labels_traefik_enabled` or `matrix_synapse_container_labels_matrix_related_labels_enabled`
+matrix_bot_draupnir_container_labels_traefik_enabled: false
+matrix_bot_draupnir_container_labels_traefik_docker_network: "{{ matrix_draupnir_bot_container_network }}"
+matrix_bot_draupnir_container_labels_traefik_hostname: "{{ matrix_synapse_container_labels_traefik_hostname }}"
+matrix_bot_draupnir_container_labels_traefik_path_prefix: "^/_matrix/client/(r0|v3)/rooms/([^/]*)/report/"
```

Comment:
Maybe it's better to rename this to `matrix_bot_draupnir_container_labels_traefik_path_regexp`, if it's going to be using regular expressions and will be passed to `PathRegexp`.

We've done the same in `matrix-media-repo`. See [here](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/33d5b0d991473d69c0a302b8684225798b247b94/roles/custom/matrix-media-repo/defaults/main.yml#L84-L85) and [here](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/33d5b0d991473d69c0a302b8684225798b247b94/roles/custom/matrix-media-repo/tasks/validate_config.yml#L32).